ecs-cli/modules/cli/cluster/cluster_app.go

```diff
@@ -11,7 +11,7 @@
 // express or implied. See the License for the specific language governing
 // permissions and limitations under the License.
 
-package command
+package cluster
```

Comment:
As for the name of the file, what do you think of calling it app.go or cli.go?
I just want us to be consistent across command and cli directories.

So either follow
```
modules/commands/cluster/command.go
modules/cli/cluster/app.go or modules/cli/cluster/cli.go
```
or
```
modules/commands/cluster/cluster_command.go
modules/cli/cluster/cluster_app.go or modules/cli/cluster/cluster_cli.go
```